data/config.yaml

```diff
@@ -21,20 +21,21 @@ main-instructions: |
         This is a dashboard to track progress of porting
         Fedora packages to Python 3.
     </div>
-    <div>
-        <br/>
+    <div style="padding-top: 7pt;">
         Read me: <a href="$HOWTO_LINK">So you want to contribute?</a><br/>
+    </div>
+    <div style="padding-top: 7pt;">
         If you would like to jump in and help port, do this:
         <ul>
             <li>
-                If you have a specific project in mind that you would like to port,
-                click on it and follow instructions there.
-                (You can use your browser – Ctrl+F – to find your package on this page.)
```

Comment:
Well, I think people capable of porting know about Ctrl+F so I thought shorter would be better.